sharktank/tests/models/llama/benchmark_amdgpu_test.py

```diff
@@ -89,136 +91,73 @@ def save_benchmarks(
             )
 
 
 @is_mi300x
 class BenchmarkLlama3_1_8B(BaseBenchmarkTest):
     def setUp(self):
-        super().setUp()
+        super().setUp(artifact_dir=Path("/shark-dev/8b"), dir_path_name="llama-8b")
         # TODO: add numpy files to Azure and download from it
-        artifact_dir = Path("/shark-dev/8b")
-        self.dir_path = self.__class__.dir_path / "llama-8b"
-        Path(self.dir_path).mkdir(parents=True, exist_ok=True)
-
-        self.llama8b_f16_torch_sdpa_artifacts = ExportArtifacts(
-            irpa_path=self.llama3_8b_f16_model,
-            batch_size=4,
-            iree_hip_target="gfx942",
-            iree_hal_target_device="hip",
-            attention_kernel="torch",
-            tensor_parallelism_size=self.tensor_parallelism_size,
-            pipeline_parallelism_size=self.pipeline_parallelism_size,
-            block_seq_stride=32,
-        )
-        self.llama8b_fp8_torch_sdpa_artifacts = ExportArtifacts(
-            irpa_path=self.llama3_8b_f8_model,
-            batch_size=4,
-            iree_hip_target="gfx942",
-            iree_hal_target_device="hip",
-            attention_kernel="torch",
-            tensor_parallelism_size=self.tensor_parallelism_size,
-            pipeline_parallelism_size=self.pipeline_parallelism_size,
-            block_seq_stride=32,
-            use_hf=True,
-            activation_dtype="bfloat16",
-            attention_dtype="bfloat16",
-            kv_cache_dtype="float8_e4m3fnuz",
-        )
-        self.llama8b_fp8_attnf8_sdpa_artifacts = ExportArtifacts(
-            irpa_path=self.llama3_8b_f8_attnf8_model,
-            batch_size=4,
-            iree_hip_target="gfx942",
-            iree_hal_target_device="hip",
-            attention_kernel="sharktank",
-            tensor_parallelism_size=self.tensor_parallelism_size,
-            pipeline_parallelism_size=self.pipeline_parallelism_size,
-            block_seq_stride=32,
-            use_hf=True,
-            activation_dtype="bfloat16",
-            attention_dtype="float8_e4m3fnuz",
-            kv_cache_dtype="float8_e4m3fnuz",
-            use_attention_mask=True,
-        )
 
-        # default fp8 input size here is 128
         self.prefill_args_fp16 = {
             128: self.save_benchmarks(
                 benchmark_fn="prefill_bs4",
-                input_path=artifact_dir / "prefill_args_bs4_128_stride_32_tp1",
-                tensor_parallelism_size=self.tensor_parallelism_size,
+                input_path=self.artifact_dir / "prefill_args_bs4_128_stride_32_tp1",
+                tensor_parallelism_size=1,
             ),
             2048: self.save_benchmarks(
                 benchmark_fn="prefill_bs4",
-                input_path=artifact_dir / "prefill_args_bs4_2048_stride_32",
-                tensor_parallelism_size=self.tensor_parallelism_size,
+                input_path=self.artifact_dir / "prefill_args_bs4_2048_stride_32",
+                tensor_parallelism_size=1,
```

Comment:
Simplifying this further is possible, but will ideally involved renaming files so they have a consistent pattern that we can create using f-strings